oxidation/libparsec_python/Cargo.toml

```diff
@@ -23,7 +23,7 @@ uuid = { version = "1.0.0-alpha.1", features = ["serde", "v4", "fast-rng"] }
 chrono = {version = "0.4.19", features = ["serde"]}
 fancy-regex = "0.8.0"
 
-[dependencies.pyo3]
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies.pyo3]
```

Comment:
libparsec_python will never be compiled for wasm32 so no need for this

Reply:
I've added a comment